app/models/form526_submission.rb

```diff
@@ -468,22 +466,11 @@ def last_remediation
   end
 
   def account
-    # first, check for an ICN on the UserAccount associated to the submission, return it if found
-    account = user_account
-    return account if account&.icn.present?
+    return user_account if user_account.icn.present?
```

Comment:
I think `if user_account.icn.present?` will throw an error is `user_account` is nil?

```
undefined method `icn' for nil (NoMethodError)
```

So I think this will need rejiggered some.